var/spack/repos/jcsda-emc-bundles/packages/ufs-pyenv/package.py

```diff
@@ -20,7 +20,7 @@ class UfsPyenv(BundlePackage):
 
     depends_on("py-cython")
     depends_on("py-cftime")
-    depends_on("py-h5py")
+    #depends_on("py-h5py")
```

Comment:
I don't see h5py getting used anywhere in UFS, at least I don't see any explicit import's.